dotcom-rendering/src/web/components/PinnedPost.tsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import {
+	neutral,
+	space,
+	textSans,
+	news,
+	from,
+	visuallyHidden,
+	transitions,
+	focusHalo,
+	height,
+} from '@guardian/source-foundations';
+import { Props, SvgMinus, SvgPlus } from '@guardian/source-react-components';
```

Comment:
What are these `Props` we're importing here? Normally when a lib exports some props they are explicitly named so you know what you're getting so it's strange to see an export called `Props`. I wonder if this was intentional...?

Reply:
@SiAdcock @sndrs - is Source meant to be exporting this?

Reply:
Think  this might have been an accident with VSC auto imports. I've removed in favour of local Props

Reply:
i _think_ that's left in from v3 because people had found it deep in the files in the package and had used it for their own components, iirc @SiAdcock? agree it should have a better name, if it should even be public...

Reply:
deprecated it here https://github.com/guardian/source/pull/1310